### ISSUE TITLE
I Get Low Accuracy When I Try To Reproduce Your IJBB / IJBC Evaluations

### ISSUE THREAD

**Issue by sourcesync** [deepinsight/insightface#2747]
Hi,

I managed to get your evaluation code working for IJBB and IJBC.

Your table claims very high accuracy, but I'm getting much lower (see below on IJBCC).

I noticed your table also says "E4" in parenthesis.

Is that a subset of the IJB datasets?

Thanks for any guidance!

George

---------

python -u ijb_1n.py --model-prefix ./w600k_r50_model_state_dict.pth --model-epoch 1 --gpu 0 --target IJBC --job arcface
...
...
processing 468500
processing 469000
img_feats (469375, 1024)
faceness_scores (469375,)
Time: 5171.64 s.
Feature Shape: (469375 , 1024) .
input features shape (469375, 512)
Finish Calculating 0 template features.
Finish Calculating 2000 template features.
Time: 3.72 s.
gallery_templates_feature (3531, 512)
gallery_unique_subject_ids (3531,)
(457519,) (457519,)
Finish Calculating 0 template features.
Finish Calculating 2000 template features.
Finish Calculating 4000 template features.
Finish Calculating 6000 template features.
Finish Calculating 8000 template features.
Finish Calculating 10000 template features.
Finish Calculating 12000 template features.
Finish Calculating 14000 template features.
Finish Calculating 16000 template features.
Finish Calculating 18000 template features.
probe_mixed_templates_feature (19593, 512)
probe_mixed_unique_subject_ids (19593,)
2025-04-13 15:37:25.700458: start evaluation
(19593, 512)
(3531, 512)
similarity shape (19593, 3531)
(19593, 3531)
top1 = 0.053029143061297405
top5 = 0.0896238452508549
top10 = 0.11340784974225489
69163290
(19593,)
neg_sims num = 69163290
after sorting , neg_sims num = 1960
far = 0.0100000000 pr = 0.0033175114 th = 0.9945181732
far = 0.1000000000 pr = 0.0089317613 th = 0.9929078339

**Comment by im-xiaoming** [comment]
Hi guy, where to find csv file like _1N_gallery_G1.csv, it did not include in IJB/C dataset that InsightFace provided.
